bigbluebutton-html5/imports/utils/deviceType.js

```diff
@@ -0,0 +1,16 @@
+const deviceType = () => {
+  // Listing of Device Viewport sizes, Updated : March 25th, 2018
+  // http://mediag.com/news/popular-screen-resolutions-designing-for-all/
+  const MAX_PHONE_SHORT_SIDE = 480;
+
+  const smallSide = window.screen.width < window.screen.height
+    ? window.screen.width
+    : window.screen.height;
+
+  return {
+    isPhone: smallSide <= MAX_PHONE_SHORT_SIDE,
```

Comment:
Checking screen w/h doenst mean its an iPhone, if you want to target only iPhone please check the `userAgent`

Reply:
"isPhone" not "iPhone"

Reply:
@capilkey LEL my bad.